### ISSUE TITLE
Change ground default color

### ISSUE THREAD

**Issue by me2beats** [godotengine/godot-proposals#6978]
### Describe the project you are working on

plugins and games

### Describe the problem or limitation you are having in your project

I find default color of the ground not so great as it could be

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

I suggest to tweak the ground color a bit

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

This is the default color
![image](https://github.com/godotengine/godot-proposals/assets/16458555/d5beffc3-b6a3-4aba-b1ee-ca1823e07908)

This one looks better to me (`#222627`)
![image](https://github.com/godotengine/godot-proposals/assets/16458555/9259df9a-8dff-4dc6-8662-85d2ec59f476)

Or this (`#262928`)
![image](https://github.com/godotengine/godot-proposals/assets/16458555/bedaa4f6-30cb-45cb-b2b0-25e032645b71)

UPD:
Or this one - closer to the default color, but still less brown (`#2f2c26`)
![image](https://github.com/godotengine/godot-proposals/assets/16458555/a372e5a1-cd63-4c7a-bd8d-830fb05c7335)

Or this (`#2c2c2d`)
![image](https://github.com/godotengine/godot-proposals/assets/16458555/33e56dcb-2290-4a6d-8dc2-7ece24d78448)

### If this enhancement will not be used often, can it be worked around with a few lines of script?

This can be worked around per project/scene - just set the ground color you like and then use that WorldEnvironment.
But the proposal is about the default look

### Is there a reason why this should be core and not an add-on in the asset library?

The proposal is about the default look of the editor

**Comment by AThousandShips** [comment]
Why is this better? You haven't given any reason, "looks better to me" isn't one. This is much less visually clear and much more cluttered or claustrophobic feeling to me

**Comment by Miskler** [comment]
The default dark color is more pleasant to work with

**Comment by Zireael07** [comment]
"More pleasant to work with" still isn't an objective reason. WHY is it more pleasant? Because of more contrast with default white?

**Comment by me2beats** [comment]
Well, yes, the perception of colors is subjective.
Therefore, I propose to choose a color by voting.

For me personally, this color looks more modern and more suitable for the editor's default theme.
Or maybe it's just because I don't like brown colors  :)

In any case, I think it's interesting to get feedback from people.

**Comment by AThousandShips** [comment]
Just making a compatibility breaking change for personal preference isn't valid, even if it's voted about

Instead I would propose adding project settings to control the default colours, making it easier to configure this

**Comment by Zireael07** [comment]
> I would propose adding project settings to control the default colours, making it easier to configure this

This please

**Comment by dalexeev** [comment]
I think there is a more generic solution that solves more than just this problem: project templates. If you can create project templates (and choose one as the default template), then you can set any initial environment that is comfortable for you, even if your preferences differ from the global defaults.

As for the ground color, I have no opinion here, I'm not a 3D user. But I don't think _slightly_ changing the default color is a major compatibility issue for a minor release (not a patch release). Of course, only if this change is supported by many users.

**Comment by me2beats** [comment]
> I think there is a more generic solution that solves more than just this problem: project templates.

Yea this would be great to have. The proposal is the is here btw https://github.com/godotengine/godot-proposals/issues/1481

**Comment by KoBeWi** [comment]
This can be implemented in a way that doesn't break compatibility, by only modifying the preview Environment. It would only affect the editor and new environments created from preview.

**Comment by YuriSizov** [comment]
This change was done on purpose to avoid tinting Godot worlds with distinct blue by default. Background contributes to the ambient light, and this current color combination, while perhaps not aesthetically pleasant, creates a more neutral environment. Which is more important to working with the engine than aesthetics since you see your imported models without deviations by default.

**Comment by yosoyfreeman** [comment]
Yeah, i used to to the same thing you did here, as is more intuitive cause you don't want to see "Ground" As part of your sky, but that said, there is a reason why most games include it in their sky or HDRI. As Yuri said, this will affect your global illumination and you don't want light on the bottom. (Technically, making it almost black would look even better).

You can test this same thing in blender. Load an HDRI, render. It looks as it should. If you do the same with one without ground is going to look messed up.

**Comment by EricDeng1001** [comment]
I would like this feature too. Godot could have a modern feel and look

**Comment by AThousandShips** [comment]
What is more modern about any of the suggested color changes?
